catalogue/webapp/pages/works.js

```diff
@@ -247,7 +244,7 @@ const Works = ({
               style={{ opacity: loading ? 0 : 1 }}
             >
               <div className="container">
-                <div className="grid">
+                <div className={isImageSearch ? 'flex flex--wrap' : 'grid'}>
```

Comment:
Could we instead pull this out and have `WorkResults` and `ImageResults`?

Reply:
There's already two conditional renderings, and assuming there will be more and the result box is fundamentally moving in a different direction.

Reply:
Yes, I think that's a good idea. Should the relevance tracking be included regardless of whether or not it's an image-only search?

Reply:
Not part of this PR - but great question, let's chat about it.